addons/alphabetical-order/userscript.js

```diff
@@ -0,0 +1,126 @@
+export default async function ({ addon, msg, console }) {
+  const vm = addon.tab.traps.vm;
+
+  Array.prototype.naturalSort = function () {
```

Comment:
Is it necessary to add this to the prototype? Doesn't sound like the best idea. Why not just make it a normal function that receives the array as an argument?

Reply:
It's not necessary but I find it much more convenient in this scenario and it has no negative effects as far as I can tell.

Reply:
Well, https://flaviocopes.com/javascript-why-not-modify-object-prototype/